internal/client.go

```diff
@@ -1082,6 +1082,9 @@ func (op *UpdateWithStartWorkflowOperation) Get(ctx context.Context) (WorkflowUp
 	case <-op.doneCh:
 		return op.handle, op.err
 	case <-ctx.Done():
+		if !op.executed.Load() {
+			return nil, fmt.Errorf("%v: %v", ctx.Err(), fmt.Errorf("operation was not executed"))
```

Comment:
for error wrapping shouldn't one of these be a `%w`?

Reply:
Indeed! I just learned that Go 1.20 supports multiple `%w`, changing it to that now.